net.js

```diff
@@ -0,0 +1,31 @@
+class Net
```

Comment:
no function update, init ?

Reply:
Not really as they were not necessary as I was writing the code, is it necessary to implement them?